packages/docs/src/styles/_sections.scss

```diff
@@ -260,6 +260,7 @@
   .pt-dialog {
     top: $content-padding;
     transform: translateX(50%);
+    z-index: inherit;
```

Comment:
i wanted it to inherit from the document flow whatever that might be, but reseting it works also.